docs/source/datasets/index.rst

```diff
@@ -461,4 +461,148 @@ The following columns are included in these datasets:
+   * - Employer ID 
+     - :code:`employer_id`
+     -  
+   * - Employer Name 
+     - :code:`employer_name`
+     - 
+   * - Employer street number
+     - :code:`employer_street_number`
+     - 
+   * - Employer street name
+     - :code:`employer_street_name`
+     - 
+   * - Employer unit number
+     - :code:`employer_unit_number`
+     - 
+   * - Employer city
+     - :code:`employer_city`    
+     - 
+   * - Employer state
+     - :code:`employer_state`  
+     - 
+   * - Employer ZIP code
+     - :code:`employer_zipcode`
+     - 
+   * - Joint filer first name
+     - :code:`spouse_first_name`
+     - Only included if the simulant has a joint filer (i.e., spouse).
+   * - Joint filer middle initial
+     - :code:`spouse_middle_initial`
+     - Only included if the simulant has a joint filer (i.e., spouse).
+   * - Joint filer last name
+     - :code:`spouse_last_name`
+     - Only included if the simulant has a joint filer (i.e., spouse).
+   * - Joint filer mailing address street number
+     - :code:`spouse_mailing_address_street_number`
+     - Only included if the simulant has a joint filer (i.e., spouse).
+   * - Joint filer mailing address street name
+     - :code:`spouse_mailing_address_street_name`
+     - Only included if the simulant has a joint filer (i.e., spouse).
+   * - Joint filer mailing address unit number
+     - :code:`spouse_mailing_address_unit_number`
+     - Only included if the simulant has a joint filer (i.e., spouse).
+   * - Joint filer mailing address PO box
+     - :code:`spouse_mailing_address_po_box`
+     - Only included if the simulant has a joint filer (i.e., spouse).
+   * - Joint filer social security number
+     - :code:`spouse_ssn`
+     - Only included if the simulant has a joint filer (i.e., spouse).
+   * - Joint filer individual tax identification number
+     - :code:`spouse_itin`
+     - Only included if the simulant has a joint filer (i.e., spouse).
+   * - Joint filer income
+     - :code:`spouse_income`
+     - Only included if the simulant has a joint filer (i.e., spouse).
+   * - Joint filer employer ID
+     - :code:`spouse_employer_id`
+     - Only included if the simulant has a joint filer (i.e., spouse).
+   * - Joint filer employer name
+     - :code:`spouse_employer_name`
+     - Only included if the simulant has a joint filer (i.e., spouse).
+   * - Joint filer employer address
+     - :code:`spouse_employer_address`
+     - Only included if the simulant has a joint filer (i.e., spouse).
+   * - Joint filer employer ZIP code
+     - :code:`spouse_employer_zipcode`
+     - Only included if the simulant has a joint filer (i.e., spouse).
+   * - Dependent 1 first name
+     - :code:`dependent_1_first_name`
+     - Only included if the simulant has a tracked dependent; to be included for each dependent on the tax filing, up to 4 dependents.
+   * - Dependent 1 middle initial
+     - :code:`dependent_1_middle_initial`
+     - Only included if the simulant has a tracked dependent.
```

Comment:
The term "tracked dependent" is specific to our simulation and won't be meaningful to pseudopeople users. As per my comment above, maybe we should just delete these notes, which would make this a moot point.